Source/Urho3D/Resource/Resource.cpp

```diff
@@ -59,8 +66,10 @@ bool Resource::Load(Deserializer& source)
     SetAsyncLoadState(ASYNC_DONE);
 
 #ifdef URHO3D_PROFILING
+#ifndef URHO3D_TRACY_PROFILING

```

Comment:
Why not single logical expression?

Reply:
#if defined (URHO3D_PROFILING) && !defined (URHO3D_TRACY_PROFILING)
...
#endif

Reply:
Do you mean why not just use a single `URHO3D_TRACY_PROFILING` to toggle this feature?

Reply:
No, I mean why are you using 4 lines for boolean condition instead of 2:

```
#ifdef
#indned
...
#endif
#endif
```

Reply:
Done!